### PR TITLE
iOS 2.2 - Cantaloupe

### PR DESCRIPTION
**Pull request by arturs105**
## User facing changelog

```
- Improved accessibility for the Time Entries Log view
- Improved suggestions
- Fixed duplicating tags issue
- Fixed back button in Forgot Password View
- Fixed syncing issues on app start
- Various stability and UX improvements
```

## Changes
https://github.com/toggl/mobileapp/compare/ios-2.1.3...release/ios-2.2

### CODE REVIEW

**Comment by juanlaube** [comment]
I'm not sure users care about the removal of onboarding steps.
Maybe collapse stability and UX improvements into one item? 
Please sort the list, i.e: first improvements, then fixes, finally other stuff.

File: Toggl.iOS/Toggl.iOS.csproj [hunk 1]
```diff
@@ -60,7 +60,7 @@
     <CodesignKey>iPhone Distribution: Toggl OU (B227VTMZ94)</CodesignKey>
     <MtouchExtraArgs>--linkskip Toggl.iOS</MtouchExtraArgs>
     <MtouchFloat32>true</MtouchFloat32>
-    <MtouchLink>Full</MtouchLink>
+    <MtouchLink>SdkOnly</MtouchLink>
```
Comment (sofokli1) [inline Toggl.iOS/Toggl.iOS.csproj]: what does this do?

Comment (arturs105) [reply Toggl.iOS/Toggl.iOS.csproj]: This is the linker setting change that fixes builds 🎊  and increases app size 😞.

Comment (sofokli1) [reply Toggl.iOS/Toggl.iOS.csproj]: Thats why apple send me an email about app size increase

**Pull request merged**